openedx/core/djangolib/markup.py

```diff
@@ -14,6 +15,14 @@
 Text = markupsafe.escape                        # pylint: disable=invalid-name
 
 
+class HTMLCleaner(Cleaner):
+    _is_url = re.compile(r"^(?:https?|ftp|file)://", re.I).search
+    def _remove_javascript_link(self, link: str):
+        if self._is_url(link.strip()):
```

Comment:
we should encapsulate this line inside `_remove_javascript_link` method
```
 _is_url = re.compile(r"^(?:https?|ftp|file)://", re.I).search
```

Reply:
I made it as a internal class method to stay close to the original code and for testing purposes. But encapsulating it should be fine as well.